src/Form/LegacyConsumer/resources/js/conditinal-fields.js

```diff
@@ -6,7 +6,7 @@ document.addEventListener('readystatechange', event => {
 	const state = {};
 
 	/**
-	 * @unreleased
+	 * @since [[UNRELEASED]]
```

Comment:
I know Jason mentioned that this is what we will want to do as convention, but the package/script isn’t updated to support this yet.

Reply:
@JasonTheAdams I thought you suggest us to use it. do you want me to replace it?

Reply:
We will be using it, but the package that we use to replace `@unreleased` with `@since <version>` needs to be updated before we adopt the new convention. Right now if we merge this, then these will not get replaced when the next release happens unless that package is updated.

Reply:
fixed: https://github.com/impress-org/givewp/compare/6e84efc7e111...078a6d08fd41

Reply:
Thanks!